src/main.rs

```diff
@@ -38,16 +41,17 @@ fn rubyfmt_file(file_path: PathBuf) -> Result<(), FileError> {
         Err(rubyfmt::RichFormatError::SyntaxError) => Err(FileError::SyntaxError),
         Err(e) => {
             // we're in a formatting loop, so print, and OK
-            handle_error_from(e, &format!("{}", file_path.display()), ErrorExit::NoExit);
+            handle_error_from(e, &file_path, ErrorExit::NoExit);
             Ok(())
         }
     }
 }
 
-fn rubyfmt_dir(path: &str) {
-    for entry in glob(&format!("{}/**/*.rb", path)).expect("it exists") {
+fn rubyfmt_dir(path: &Path) {
+    // FIXME: Look for an implementation of glob that actually takes a proper base dir
```

Comment:
do we have a corresponding issue for this?

Reply:
#308